Dockerfile

```diff
@@ -180,25 +186,33 @@ RUN bash ./download-chrome.sh
 COPY scripts/install-mono.sh .
 RUN bash ./install-mono.sh
 
+# TODO: checksum
 # Install Elixir
 ENV MIX_HOME=/home/cukebot/.mix
-RUN curl -SL --output erlang.deb https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb \
+RUN curl -sSL https://packages.erlang-solutions.com/erlang-solutions_2.0_all.deb -o erlang.deb \
+    && echo "1968ec2ae81a5e1f56d2f173144926ec90a5e7c7  erlang.deb" | sha1sum -c --quiet - \ 
     && dpkg -i erlang.deb \
-    && rm -f erlang.deb \
     && apt-get update \
     && apt-get install --assume-yes --no-install-recommends \
     esl-erlang \
     elixir \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f erlang.deb
 
 USER $USER
 
+# TODO: checksum
 ## As a user install node and npm via node version-manager
-RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash \
+WORKDIR /home/$USER
+RUN curl https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh -o install-nvm.sh \
```

Comment:
This command uses a different set of options from the others. Why?

Reply:
I missed it, that's all.